src/components/elements/list/types.ts

```diff
@@ -1,20 +1,40 @@
 import { V2_OrderedListProps, V2_UnorderedListProps } from "@lifesg/react-design-system/v2_text-list";
+import { OrderedListProps, UnorderedListProps } from "@lifesg/react-design-system/text-list";
 import { TComponentOmitProps, TFrontendEngineFieldSchema } from "../../frontend-engine";
 import { IBaseElementSchema } from "../types";
 import { TWrapperSchema } from "../wrapper";
 
-export interface IUnorderedListSchema<V = undefined, C = undefined>
+/**
+ * @deprecated Use design system v3 IUnorderedListSchema
+ */
+export interface IV2UnorderedListSchema<V = undefined, C = undefined>
```

Comment:
is there a need for the separate interface? could you `Omit<..., "size">` and redeclare the size prop to be `V2_TextSizeType | TypographySizeType`?

Reply:
to clarify, I meant if `IV2OrderedListSchema` could be merged into `IOrderedListSchema`, and `IV2UnorderedListSchema` into `IUnorderedListSchema` instead of introducing separate interfaces